src/app/pages/signup/signup.component.html

```diff
@@ -8,7 +8,7 @@ <h1>Inscription</h1>
   <form [formGroup]="signupForm" (ngSubmit)="onSubmit()">
     <div class="form-group">
       <label for="firstName">Prénom</label>
-      <input id="firstName" formControlName="firstName" type="text" required="" />
+      <input id="firstName" formControlName="firstName" type="text" required=""  placeholder=""/>
```

Comment:
if you are using placeholder attribute, fill it with placeholder string

Reply:
J'ai oublié d'effacer just pour le test

Reply:
I forgot to delete just for the test